docs/io/visualization/sdec_plot.ipynb

```diff
@@ -328,7 +328,7 @@
    "source": [
```

Comment:
<p>There's an extra newline here that shouldn't be present</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1597/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>